### PR TITLE
Only shorten blcoking timeouts for non-quorum commands

### PR DESCRIPTION
**Pull request by tylerkaraszewski**
### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/418167

### Tests

_________
**Internal Testing Reminder:** when changing bedrock, please compile auth against your new changes

### CODE REVIEW

File: BedrockCore.cpp [hunk 1]
```diff
@@ -135,7 +135,7 @@ BedrockCore::RESULT BedrockCore::peekCommand(unique_ptr<BedrockCommand>& command
             if (!_db.beginTransaction(exclusive ? SQLite::TRANSACTION_TYPE::EXCLUSIVE : SQLite::TRANSACTION_TYPE::SHARED)) {
                 STHROW("501 Failed to begin " + (exclusive ? "exclusive"s : "shared"s) + " transaction");
             }
-            if (exclusive) {
+            if (exclusive && command->writeConsistency != SQLiteNode::QUORUM) {
```
Comment (aldo-expensify) [inline BedrockCore.cpp:138]: Do we also need this check in `processCommand`? (the other place that can change the timeout)

**Comment by aldo-expensify** [comment]
I'm not familiar with "quorum commands", is `BatchReimbursements` one of them?

File: BedrockCore.cpp (hunk 1 above)
Comment (tylerkaraszewski) [reply BedrockCore.cpp:138]: Yes, adding.

**Comment by tylerkaraszewski** [comment]
QUORUM commands are defined by this list:
https://github.com/Expensify/Bedrock/blob/main/BedrockServer.cpp#L1271

Which is currently set as:
```
-synchronousCommands AdminCancelReimbursement,BatchReimbursements,CancelReimbursement
```

**Review by aldo-expensify — APPROVED** [review]

**Pull request merged**